server/spec/autobind_disabled_for_owner_spec.rb

```diff
@@ -85,10 +85,14 @@
   end
 
   it 'will still register when content access setting enabled and autobind enabled on activation key' do
+    # org_environment mode cannot be used in standalone mode
+    skip("candlepin running in standalone mode") if not is_hosted?
```

Comment:
I'll go back and take a look at this more closely, but in general, the places that relied on setting the org_environment during creation were somewhat broken on the CP side, and the new restrictions renders the tests broken in standalone mode. I'll get into more details with the comment below in OwnerResource.

Reply:
Looking at the tests again, both of them attempt to use org_environment as the operating CA mode, which is what got the tests disabled.

Setting the list is still possible in standalone mode during creation, but it's the reliance on the mode change which is not.

Reply:
Alright, it's just that the comment is misleading when it says "org_environment mode cannot be used in standalone mode". An org can get into org_environment mode through the import process, while someone new to the code looking at this comment might think that org_environment is a hosted-only feature (I certainly would think so, since I always looked to the spec tests to find out more about use cases of features). Maybe It would be more accurate to change this comment (and other similar ones in the spec tests, where appropriate) to "org_environment mode cannot be _**set directly**_ in standalone mode"?

Reply:
I've updated the comment above the skip on these tests to read "org_environment mode cannot be set via API in standalone mode"